docs/installation/installation.md

```diff
@@ -1,59 +1,64 @@
-# tyk-operator installation
+# Tyk Operator Installation
 
 - [Prerequisites](#prerequisites)
-- [Installing Tyk](#installing-tyk)
-- [Operator Configuration](#tyk-operator-configuration)
-- [Installing CRDs](#installing-crds)
-- [Installing cert-manager](#installing-cert-manager)
-- [Installing tyk-operator](#installing-tyk-operator)
+- [Installing Tyk Gateway](#installing-tyk-gateway)
+- [Tyk Operator Configuration](#tyk-operator-configuration)
+  - [Connecting Tyk Operator to Tyk Gateway](#connecting-tyk-operator-to-tyk-gateway)
+    - [Watching Namespaces](#watching-namespaces)
+    - [Watching custom ingress class](#watching-custom-ingress-class)
+  - [Installing CRDs](#installing-crds)
+  - [Installing cert-manager](#installing-cert-manager)
+- [Installing Tyk Operator](#installing-tyk-operator)
```

Comment:
Watching Namespaces and Watching custom ingress class should not be nested inside `Connecting Tyk Operator to Tyk Gateway`. They should be at the same level.

I think `[Installing CRDs]` should belong just before `[Installing Tyk Operator]`. And they need to be side-by side.

Reply:
This TOC reflects the document itself; it's just based on the headings in the file and their sequence there. I'll rearrange and move CRDs. 👍

Reply:
5a15d8f and 1c13e9c should resolve